lib/misc.js

```diff
@@ -19,6 +19,20 @@ module.exports = (api) => {
     return callback ? wrap : api.common.falseness;
   };
 
+  api.common.cbExtract = (
+    // Exctracts callback and wraps it with api.common.cb
+    // callback is last argument if it's function
+    // otherwise it's api.common.falseness
+    args // arguments
+    // Returns wrapped callback
+  ) => {
+    const argsLast = args[args.length - 1];
```

Comment:
@tshemsedinov I've been thinking about that name and have decided to name `argsLast` because it's last of arguments (which `restLeft` receives) for sure but it can be either callback or last of arguments (which wrapped function receives). Therefore `cb` would be false name if no callback was passed to function.
But I will try to use `cb` name.